website/docs/d/cloudant.html.markdown

```diff
@@ -0,0 +1,61 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_cloudant"
+description: |-
+  Get information about Cloudant instance.
+subcategory: "Cloud Databases"
+---
+
+# ibm_cloudant
+
+Provides a read-only data source for an existing IBM Cloud Cloudant service. You can then reference the fields of the data source in other resources within the same configuration using interpolation syntax.
+
+## Example Usage
+
+```hcl
+resource "ibm_resource_instance" "cloudant" {
+    name     = "cloudant-service-name"
+    service  = "cloudantnosqldb"
+    plan     = "lite"
+    location = "us-south"
+ }
+
+ data "ibm_cloudant" "instance" {
+    name     = ibm_resource_instance.cloudant.name
+ }
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `id` (Optional, String) The unique identifier of the Cloudant resource.
+* `location` (Optional, String) The location or the environment in which instance exists.
```

Comment:
The location or the environment in which instance exists.  should be as
The location or the environment in which the instance exists.